disasterinfosite/models.py

```diff
@@ -333,80 +333,89 @@ def getRelatedTemplate(self):
     def findSnuggetsForPoint(lat=0, lng=0, merge_deform = True):
         pnt = Point(lng, lat)
 
-######################################################
-# GENERATED CODE GOES HERE
-# DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
-# modelsGeoFilters
```

Comment:
Replacing the generated code block means we won't be able to use import.py on this again.  If this is deliberate, then go ahead and merge this, but I'm a little wary because we still have a few shapefiles with "placeholder" in the name.  We can probably deal with their replacements by either saving the new ones with the same name or doing a find-and-replace in the code, but if the field names change it will be some hassle.

Of course, we'll still be able to use the rest of the automated pipeline, as long as we manually do the equivalent of the code changes import.py would have made.